tzrec/tests/rank_integration_test.py

```diff
@@ -70,18 +70,37 @@ def _test_rank_nofg(self, pipeline_config_path, reserved_columns, output_columns
 
     @unittest.skipIf(not torch.cuda.is_available(), "cuda not found")
     def test_aot_export(self):
```

Comment:
trt should not use multi-tower-din trt config, should use multi-tower-din config now

Reply:
global trt convert is not support yet